### ISSUE TITLE
API should not increase install count

### ISSUE THREAD

**Issue by jasonlewis** [UserScape/Laravel-Bundles#53]
I'm going to look into this but I think it'd be better if a call to the API did not increase the install count for that bundle. I'm wanting to use the API for a bundle of mine but just noticed from looking here that every call I make to the API will result in a +1 to the install count for that bundle.

Perhaps have a flag that Artisan can set in the URI that enabled the increment or use an entirely different route. I'm not sure what you'd like to see here.

**Comment by ericlbarnes** [comment]
I think it would be nice to totally rename that route. But since that really isn't an option I guess the second best way would be to create one for access outside artisan. 

Speaking of the api do you need anything else? Or just the bundle info?

**Comment by jasonlewis** [comment]
Yeah can't really rename it now. Or you could, you'd just have to force everyone to update their artisan related things.

As for anything else... At the moment I just fetched the bundle information and then from there fetched further details from GitHub itself. What else could you provide? User details perhaps?

**Comment by franzliedke** [comment]
Are there any plans to fetch the download count (and maybe use that for popularity measurement) via the GitHub API?
